.licenserc.yaml

```diff
@@ -0,0 +1,20 @@
+# See: https://github.com/apache/skywalking-eyes
+header:
+  license:
+    spdx-id: GPL-3.0-or-later
+    # This is the pattern used when running:
+    #   docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header check
+    pattern: |
+      Copyright(?: \(c\))? \b(19\d{2}|20\d{2})\b The (VeChainThor developers|go-ethereum Authors)
```

Comment:
This is checking for the first line only. We can optionally add in the rest, but it will probably be too hard to digest as the ethereum pattern is longer & different to vechain